configs/gooddata.json

```
@@ -3,6 +3,12 @@
     "start_urls": [
         "https://sdk.gooddata.com/gooddata-ui/docs/",
         "https://sdk.gooddata.com/gooddata-ui/docs/about_gooddataui.html",
+        {
+            "url": "https://sdk.gooddata.com/gooddata-ui-apidocs/v(?P<version>.*?)",
```

```suggestion
            "url": "https://sdk.gooddata.com/gooddata-ui-apidocs/v(?P<version>.*?)/",
```

(to avoid a warning in the crawler from missing / after regexes)